### PR TITLE
Fix splash screen issue

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**

### CODE REVIEW

**Comment by gguijarro-c-chwy** [comment]
I tested this on IOS and it works.

**Comment by gguijarro-c-chwy** [comment]
i should say, on an Old IOS device :)

File: ios/Podfile.lock [hunk 1]
```diff
@@ -1067,4 +1067,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d60f8e2e421d286a4337134adead709daf32f131
```
Comment (n13) [inline ios/Podfile.lock:1069]: This is probably unintentional

**Review by n13 — APPROVED** [review]

**Comment by n13** [comment]
reverted the change to podfile lock

**Comment by n13** [comment]
tested fix, works well, merging

**Pull request merged**